internal/client/client.go

```diff
@@ -38,7 +38,7 @@ import (
 	"go.temporal.io/sdk/client"
 )
 
-var TemporalCloudAPIVersion = "2024-05-13-00"
+var TemporalCloudAPIVersion = "2024-10-01-00"
```

Comment:
does a summary of changes for this version, besides enums, exist to determine if any behavior or breaking changes  will impact TF users?

Reply:
ack - TY.